vips/vip-322/bscmainnet.ts

```diff
@@ -0,0 +1,128 @@
+import { ethers } from "ethers";
+import { parseUnits } from "ethers/lib/utils";
+
+import { ProposalType } from "../../src/types";
+import { makeProposal } from "../../src/utils";
+
+export const XVS_BRIDGE_SRC = "0xf8F46791E3dB29a029Ec6c9d946226f3c613e854";
+export const NORMAL_TIMELOCK = "0x939bD8d64c0A9583A7Dcea9933f7b21697ab6396";
+export const COMPTROLLER = "0xfD36E2c2a6789Db23113685031d7F16329158384";
+export const BSC_TREASURY = "0xF322942f644A996A617BD29c16bd7d231d9F35E9";
+export const ETH_TREASURY = "0xFD9B071168bC27DBE16406eC3Aba050Ce8Eb22FA";
+export const XVS = "0xcF6BB5389c92Bdda8a3747Ddb454cB7a64626C63";
+
+export const REWARDS = [
+  {
+    market: "WETH",
+    reward: parseUnits("1125", 18),
+  },
+  {
+    market: "WBTC",
+    reward: parseUnits("3375", 18),
+  },
+  {
+    market: "USDT",
+    reward: parseUnits("3375", 18),
+  },
+  {
+    market: "USDC",
+    reward: parseUnits("3375", 18),
+  },
+  {
+    market: "crvUSD",
+    reward: parseUnits("1500", 18),
+  },
+  {
+    market: "FRAX",
+    reward: parseUnits("600", 18),
+  },
+  {
+    market: "sFRAX",
+    reward: parseUnits("600", 18),
+  },
+  {
+    market: "TUSD",
+    reward: parseUnits("200", 18),
+  },
+  {
+    market: "DAI",
+    reward: parseUnits("500", 18),
+  },
+  {
+    market: "CRV",
+    reward: parseUnits("375", 18),
+  },
+  {
+    market: "crvUSD",
+    reward: parseUnits("375", 18),
+  },
+  {
+    market: "WETH",
+    reward: parseUnits("18333", 18),
+  },
+  {
+    market: "wstETH",
+    reward: parseUnits("3600", 18),
+  },
+];
+
+export const BRIDGE_FEES = parseUnits("0.1", 18);
```

Comment:
I would set 0.2. The not used BNB will be sent back to the Treasury, so it should be safe. With a gas cost of 45 gwei on Ethereum, a bridge would cost 0.11 BNB, for example.